packages/shared/components/molecules/AssetIcon.svelte

```diff
@@ -39,9 +39,7 @@
             icon = ''
     }
 
-    $: shouldShowBadge = showVerifiedBadgeOnly
-        ? asset?.verification?.verified
-        : asset?.verification?.status !== NotVerifiedStatus.Skipped
+    $: shouldShowBadge = asset?.metadata.standard !== TokenStandard.BaseToken
```

Comment:
It should always show the badge, because it's possible that you have the base token on L2 or on a testnet.